ot3_state_manager/pyproject.toml

```diff
@@ -23,13 +24,15 @@ isort = "5.10.1"
 mdformat-gfm = "^0.3.5"
 mdformat_frontmatter = "^0.4.1"
 mdformat_footnote = "^0.1.1"
+pydantic = "1.8.2"
```

Comment:
what are we using pydantic for? it doesn't seem to be used anywhere

Reply:
Thank you. I was going to use it for the message format but changed my mind. I will remove it